test/TestServerProjectsLowLevel/Directory.Build.targets

```diff
@@ -0,0 +1,10 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
There are a number of compiles of this file, can we create one and relative include it to reduce duplication?

Reply:
@m-nash and I had a chat -- we'll try to refactor this out into a common file -- if we run into issues we'll go ahead with the changes as-is and we'll follow-up when @chamons and @pakrym are back 🙂.